ui/components/app/srp-input/srp-input.js

```diff
@@ -19,6 +19,10 @@ const { isValidMnemonic } = ethers.utils;
 
 const defaultNumberOfWords = 12;
 
+const isLowerCase = (draftSrp) => {
```

Comment:
Sorry for not noticing the first time but should this be "hasUpperCase"?  I think `isLowerCase` would be a good thing in this case.

Reply:
done